site/public/js/server.js

```diff
@@ -458,6 +458,7 @@ function editFilePathRecommendations() {
 
 var lastActiveElement = null;
 function captureTabInModal(formName, resetFocus=true){
+    $('body').css('overflow', 'hidden');
```

Comment:
This change shouldn't be included in the captureTabInModal() function as the function is only used within some modals that require scrolling, this is the reason why you had to include the same line again in the for the release all form.